atramhasis/templates/tree.jinja2

```diff
@@ -99,28 +100,30 @@
 
       // A recursive helper function for performing some setup by walking through all nodes
 
-      function visit(parent, visitFn, childrenFn) {
+      function visit(parent, visitFn, childrenFn, depth) {
+        depth = depth || 0;
```

Comment:
Kunnen we die depth niet buiten de functies zelf bepalen ipv hier een check op te zetten?